android/src/newarch/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -181,6 +181,11 @@ public void setInjectedJavaScriptBeforeContentLoadedForMainFrameOnly(RNCWebView
 
     }
 
+    @ReactProp(name = "injectJavaScriptObject")
+    public void setInjectJavaScriptObject(RNCWebView view, @Nullable String value) {
```

Comment:
Probably should be _injected_ rather than _inject_, since this is a property, not a function